.github/workflows/send-screenshots-to-pixeleagle.yml

```diff
@@ -16,18 +16,29 @@ on:
         required: true
         type: string
 
+env:
+  # Unfortunately, we can't check secrets in `if:` conditionals. However, Github's own documentation
+  # suggests a workaround: Putting the secret in an environment variable, and checking that instead.
+  PIXELEAGLE_TOKEN_EXISTS: ${{ secrets.PIXELEAGLE_TOKEN != '' }}
+
 jobs:
   send-to-pixel-eagle:
     name: Send screenshots to Pixel Eagle
     runs-on: ubuntu-24.04
     steps:
```

Comment:
```suggestion
  send-to-pixel-eagle:
    name: Send screenshots to Pixel Eagle
    runs-on: ubuntu-24.04
    if: ${{ github.repository == "bevyengine/bevy" }}
    steps:
```

Reply:
So, I'm not actually sure whether to commit this suggestion.

My intention is to allow forks to run the Pixel Eagle job if they desire. That's the reason I chose to lock the Pixel Eagle job behind needing a token, rather than behind what repository it's on.

However, after having slept between sending this PR and now, I'm not actually sure there's any reason a fork would want to run this.

That said, I'm also not sure why we would lock it solely to the Bevy repository.

@BD103 What do you think?

Reply:
So, I'm going to apply this suggestion - but I'm going to reword the comment first, as I don't think it fits with my intention to allow the Pixel Eagle job to run on forks.

Reply:
I think the environment variable was a superior solution because it allows forks to set that up for themselves, but checking the name of the repository is the existing pattern we use in other context.

Reply:
I think the environment variable was a superior solution because it allows forks to set that up for themselves, but checking the name of the repository is the existing pattern we use in other contexts.